integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -611,13 +611,16 @@ func TestVRFv2PlusMultipleSendingKeys(t *testing.T) {
 	t.Parallel()
 	l := logging.GetTestLogger(t)
 
+	network, err := actions.EthereumNetworkConfigFromEnvOrDefault(l)
+	require.NoError(t, err, "Error building ethereum network config")
+
 	var vrfv2PlusConfig vrfv2plus_config.VRFV2PlusConfig
-	err := envconfig.Process("VRFV2PLUS", &vrfv2PlusConfig)
+	err = envconfig.Process("VRFV2PLUS", &vrfv2PlusConfig)
 	require.NoError(t, err)
 
 	env, err := test_env.NewCLTestEnvBuilder().
 		WithTestInstance(t).
-		WithGeth().
+		WithPrivateEthereumNetwork(network).
```

Comment:
So we are using the PoS network for these tests after all? Or does this change imply something else?

Reply:
if `ETH2_EL_CLIENT`  env var is not set, then it will use PoW with Geth by default

[code](https://github.com/smartcontractkit/chainlink/blob/cf74cd0c9e9601c213b03c9d076339464a634291/integration-tests/actions/private_network.go#L23-L30)